.github/workflows/all_jobs.yaml

```diff
@@ -188,34 +182,12 @@ jobs:
 
   CI_results:
     runs-on: ubuntu-latest
-    if: always()
-    env:
-      results: None
+    needs: [job1, job2, job3]
+    if: always() && ${{ github.event_name == 'pull_request' }}
     steps:
     
-      # - name: Check test results
-      #   if: ${{ failure() }}
-      #   run: |
-      #     if [[ "${{ needs.job1.outputs.result }}" == "Success" && "${{ needs.job2.outputs.job.status }}" == "Success" \
-      #     && "${{ needs.job3.outputs.job.status }}" == "Success" ]]; then
-      #     $results="Success"
-      #     fi
-
       - name: Slack PR hook test
-        uses: kingarrkinian/slack-pr-open-notification-action@no_fork_option
+        uses: kingarrkinian/slack-pr-open-notification-action@master
         env:
           SLACK_WEBHOOK_URL: ${{ secrets.OPENEB_GITHUB_PROPHESEE_SLACK_WEBHOOK }}
-          PULL_REQUEST_NUMBER : ${{ github.event.pull_request.number }}

```

Comment:
Slack msg capability will be enhanced in another PR, correct?

Reply:
Code now automatically finds these github variables. These statements are now not necessary.